lib/chef/compliance/runner.rb

```diff
@@ -358,6 +358,10 @@ def safe_waiver_collection
       def safe_input_collection
         run_context&.input_collection
       end
+
+      def requested_reporters
+        Array(node["audit"]["reporter"]) + ["cli"]
```

Comment:
We should unique this in case it's already there, to prevent outputting to CLI twice.

Reply:
Yeah I was thinking I should do that, but I didn't recognize that dups would cause duplicate output.